### DISCUSSION TITLE
Add option to prevent closing specific folder(s)

### DISCUSSION THREAD

**Discussion by explor202** [LesFerch/WinSetView#139] [Ideas]
Tonight I finally took the time to learn more about how WinSetView works, and it's truly **awesome**! One feature I would really like to see is an option to help prevent accidental closing of specific parent folders that are currently open. For example, I routinely have 24 folders open at one time, then will open several child folders. If I get a little carried away closing the child folders, I can sometimes accidentally close the parent folder too, which then messes up the order of the other folders, forcing me to close and reopen all of them (yes, I know, it's an issue with me!!!). Is there any way an option could be added to add a list of parent folders to WinSetView, so that if "X" is clicked, some type of dialog box would display that says "Are you sure you want to close this folder?" with Y/N options before actually closing it?

**Comment by LesFerch** [comment]
Short answer: No

Long answer: WinSetView does not install any process. It simply sets registry values that Explorer already supports. Since Explorer has no option for what you describe, there is nothing that WinSetView can do. A program such as [QTTabBar](http://qttabbar.wikidot.com/), which extends Explorer's capabilities, would be the kind of program where the developer might be able to add a feature like that.

**Reply by explor202** [reply]
> Short answer: No
> 
> Long answer: WinSetView does not install any process. It simply sets registry values that Explorer already supports. Since Explorer has no option for what you describe, there is nothing that WinSetView can do. A program such as [QTTabBar](http://qttabbar.wikidot.com/), which extends Explorer's capabilities, would be the kind of program where the developer might be able to add a feature like that.

Got it, and thanks for the pointer.
